config.json

```
@@ -356,13 +356,26 @@
       "slug": "expeditions-elixir-and-exercism",
       "title": "Expeditions, Elixir and Exercism",
       "blurb": "In this week's Community Story, Brian and Jonathan take a moment to chat about cross-cultural experience, functional programming and what it was like to move from the American mid-west to Scandinavia.",
-      "published_at": "2022-11-27 12:01",
+      "published_at": "2022-12-02 17:01",
       "length_in_minutes": 45,
       "thumbnail_url": "https://exercism-static.s3.eu-west-1.amazonaws.com/blog/story-brian-underwood.jpg",
       "image_url": "https://test.org/image.png",
       "interviewer_handle": "jonathandmiddleton",
       "interviewee_handle": "cheerfulstoic",
       "youtube_id": "X_oTsaLzSck"
+    },
+    {
+      "uuid": "bfef3d25-7942-4b6d-877b-c970e445cc9d",
+      "slug": "entrepreneurial-by-design",
+      "title": "Entrepreneurial by Design",
+      "blurb": "Jonathan and Taiyab approach things from a different angle in this weeks in Community Story. For all those entrepreneurs out there looking to dabble in tech, Taiyab sheds some light on ways to see things from the business and product design perspective.",
```

```suggestion
      "blurb": "Jonathan and Taiyab approach things from a different angle in this week's Community Story. For all those entrepreneurs out there looking to dabble in tech, Taiyab sheds some light on ways to see things from the business and product design perspective.",
```

---

Maybe this would catch people's attention more?
```suggestion
      "blurb": "Are you an entrepreneur looking to dabble in tech? Taiyab sheds some light on ways to see things from the business and product design perspective in this week's Community Story.",
```

---

@jonathandmiddleton DId you see the second suggestion too? I don't know if that's the correct actual wording but rather than making this a description, making it something that will immediately make people want to listen after the first couple of words?

---

@iHiD yep that makes a lot of sense. Can't seem to update on my side but happy to proceed.